src/prommis/solvent_extraction/solvent_extraction.py

```diff
@@ -106,21 +115,8 @@ def build(self):
         )
 
         self.partition_coefficient = Param(
+            self.mscontactor.elements,
             self.mscontactor.stream_component_interactions,
-            initialize={
-                ("aqueous", "organic", "Al"): 3.6 / 100,
-                ("aqueous", "organic", "Ca"): 3.7 / 100,
-                ("aqueous", "organic", "Fe"): 2.1 / 100,
-                ("aqueous", "organic", "Sc"): 99.9 / 100,
-                ("aqueous", "organic", "Y"): 99.9 / 100,
-                ("aqueous", "organic", "La"): 75.2 / 100,
-                ("aqueous", "organic", "Ce"): 95.7 / 100,
-                ("aqueous", "organic", "Pr"): 96.5 / 100,
-                ("aqueous", "organic", "Nd"): 99.2 / 100,
-                ("aqueous", "organic", "Sm"): 99.9 / 100,
-                ("aqueous", "organic", "Gd"): 98.6 / 100,
-                ("aqueous", "organic", "Dy"): 99.9 / 100,
-            },
             mutable=True,
```

Comment:
I think you should assign a default value here so that these are not left empty. I would suggest either a value of 1 or a small positive value.

Reply:
I have added 1 as a default value for the coefficients